### PR TITLE
LIR: search stations by name or partial name

### PR DESCRIPTION
**Pull request by vasile**

### CODE REVIEW

File: Sources/OJP/OJPHelpers.swift [hunk 1]
```diff
@@ -77,30 +80,36 @@ enum OJPHelpers {
...
-
-        // TODO: - move them in SDK?
-        let requestXMLRootAttributes = [
-            "xmlns": "http://www.vdv.de/ojp",
-            "xmlns:siri": "http://www.siri.org.uk/siri",
-            "xmlns:xsi": "http://www.w3.org/2001/XMLSchema-instance",
-            "version": "2.0",
-        ]
-
+    static func buildXMLRequest(ojpRequest: OJPv2) throws -> String {
```
Comment (r3to) [inline Sources/OJP/OJPHelpers.swift:108]: → `buildXML(ojpRequest: OJPv2)`

**Review by r3to — APPROVED** [review]
lgtm

**Comment by terencebls** [comment]
If we make a search (on beta) with "ba", nothing comes 🐙

**Comment by vasile** [comment]
> If we make a search (on beta) with "ba", nothing comes 🐙 Any ideas?

I can confirm that `Ba` returns 0 results. But `Be`, `Bi` works. 
For `Ba` would you expect stops like `Bahnhof` ?

**Comment by terencebls** [comment]
> > If we make a search (on beta) with "ba", nothing comes 🐙 Any ideas?
> 
> I can confirm that `Ba` returns 0 results. But `Be`, `Bi` works. For `Ba` would you expect stops like `Bahnhof` ?

I would expect:

Baden
Basel
... something like that :)

**Review by terencebls — APPROVED** [review]

**Pull request merged**